src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py

```diff
@@ -851,7 +851,7 @@ def setUp(self):
         self.cmd('extension add -n eventgrid')
 
     def tearDown(self):
-        self.cmd('extension remove -n eventgrid')
+        # self.cmd('extension remove -n eventgrid')
```

Comment:
Any specific reason for this change which caused #17434?

Reply:
We should not suppose to remove extension after test done to avoid other test when parallel run. For example both `NetworkVirtualApplianceScenarioTest` and `NetworkSecurityPartnerProviderScenarioTest` rely on `virtual-wan` extension. If the first one remove extension when tearDown, it will cause error for unfinished one. We encounter this issue before.

Reply:
You need `@serial_test()`